app/controller/articles-controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectArticle } = require("../model/select-article");
+
+exports.getArticle = (req, res, next) => {
+  const { article_id } = req.params;
+
+  selectArticle(article_id)
+    .then((result) => {
+      res.status(200).send({ article: result });
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
Controller's looking great, though could be improved by naming the variable `result` a bit more clearly